src/LiveComponent/composer.json

```diff
@@ -40,7 +40,8 @@
         "doctrine/doctrine-bundle": "^2.0",
         "doctrine/orm": "^2.7",
         "zenstruck/foundry": "^1.10",
-        "zenstruck/browser": "^0.5.0"
+        "zenstruck/browser": "^0.5.0",
+        "doctrine/annotations": "^1.0"
```

Comment:
Was this intentional?

Reply:
Unfortunately, yes. The newest version of doctrine (bundle or orm, not quite sure) doesn't require annotations. The lowest version we currently support doesn't allow attributes so I was stuck...

I didn't look too deep into the issue but this was an easy fix. I'm open to ideas!

Reply:
That's fine then - this is just for running some functional/integration test that contain a Doctrine entity (and we use annotations for those to function on all supported DoctrineBundle versions)?